test/valid-packages-test.js

```diff
@@ -306,7 +306,26 @@ packages.map(function(pkg) {
                 pkgName(pkg) + ": Need to remove \"/\" at the front or the last of basePath in package.json");
         }
       }
-  };
+  }
+  packageVows[pname + ": Can't has duplicated keywords"] = function(pkg) {
+      var json = parse(pkg, true);
+      var kw = json.keywords;
+      var keywordsList = [];
+      var flag = 0;
+      if (kw !== undefined) {
+        for (var i = 0; i < kw.length; i++) {
+          for (var j = i+1; j < kw.length; j++) {
+            if (kw[i] === kw[j]) {
+                flag = 1;
+                if (keywordsList.indexOf(kw[j]) < 0) keywordsList.push(kw[i]);
+            }
+          }
+        }
```

Comment:
Please try to write this part more elegant like this example:
https://github.com/jsifalda/filter-duplicates/blob/master/index.js
```js
function arrayUnique (array) {
  return array.filter(function (item, position) {
    return array.indexOf(item) === position
  })
}
```